app/cypress/integration/index.spec.js

```diff
@@ -10,50 +11,597 @@ context('Homepage', () => {
 
     cy.get('h1').contains('Welcome');
 
-    // cy.get('main')
-    //   .contains('Login')
-    //   .click()
-    //   .then(() => {
-    //     cy.visit('/form/1');
-    //     cy.get('h1').contains('Organization Profile');
-    //   });
-
-    // cy.get('[id="root_projectTitle"]').type('test');
-
-    // cy.get('input[id="root_typeOfOrganization-0"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-1"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-2"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-3"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-4"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-5"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-6"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-7"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-8"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-9"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-10"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-11"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-12"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-13"]').parent().click();
-    // cy.get('input[id="root_typeOfOrganization-14"]').parent().click();
-
-    // cy.get('[id="root_organizationName"]').type('test');
-
-    // cy.get('[id="root_isLegalPrimaryName-1"]');
-
-    // cy.get('button')
-    //   .contains('Continue')
-    //   .click()
-    //   .then(() => {
-    //     cy.get('h1').contains('Organization location');
-    //   });
-
-    // cy.get('body').happoScreenshot();
+    // Todo: find a way around using these wait
+    cy.wait(2000);
+
+    cy.get('button').contains('Go to dashboard').should('be.visible').click();
+
+    cy.waitUntil(() => cy.url().should('contain', '/dashboard'));
```

Comment:
```suggestion
    cy.url().should('contain', '/dashboard');
```
This should do the same thing, no? Cypress will retry the assertion for a while, until it is true